c2cgeoportal/scaffolds/update/CONST_Makefile_tmpl

```diff
@@ -419,8 +419,8 @@ serve: build development.ini
 	$(VENV_BIN)/pserve --reload --monitor-restart development.ini
 
 .PHONY: update-node-modules
-update-node-modules:
-	npm install --force
+update-node-modules: node_modules/.bin/npm
+	node_modules/.bin/npm install --force
```

Comment:
To have the latest npm release, this is needed because of that: https://github.com/nodejs/nan/issues/414